### ISSUE TITLE
Yoem's bounded loss function

### ISSUE THREAD

**Issue by simonrnss** [AI-SDC/AI-SDC#92]
In [Yoem et al 2018](https://arxiv.org/abs/1709.01604v5) they show an attack vector they call the "bounded loss function". In it's simplest form (with 0, 1) loss, this is equivalent to predicting that:
1. Examples where the target classifier is correct belong to the training set
2. Example where the target classifier is wrong do not belong to the training set

Where a model has been overfitted, this can be quite successful.

Suggestion here is that we modify the code such that we can provide the performance of this kind of attack into the metrics, as something of a baseline.

**Comment by jim-smith** [comment]
good plan
